lib/create_workout/set_timings.dart

```diff
@@ -87,33 +91,48 @@ class _SetTimingsState extends State<SetTimings> {
       children: [
         Center(
           child: Padding(
-            padding: const EdgeInsets.fromLTRB(0.0, 20.0, 0.0, 0.0),
-            child: Row(
-              mainAxisAlignment: MainAxisAlignment.center,
-              children: [
+              padding: const EdgeInsets.fromLTRB(0.0, 20.0, 0.0, 0.0),
+              child:
+                  Row(mainAxisAlignment: MainAxisAlignment.center, children: [
                 const Icon(Icons.fitness_center),
-                IconButton(
-                  key: const Key('work-decrement'),
-                  icon: const Icon(Icons.remove),
-                  onPressed: () => setState(() {
-                    final newValue = _exerciseTime - 1;
-                    _exerciseTime = newValue.clamp(1, 120);
-                    _exerciseChanged = true;
-                  }),
-                ),
-                Text('Working time: $_exerciseTime seconds'),
-                IconButton(
-                  key: const Key('work-increment'),
-                  icon: const Icon(Icons.add),
-                  onPressed: () => setState(() {
-                    final newValue = _exerciseTime + 1;
-                    _exerciseTime = newValue.clamp(1, 120);
-                    _exerciseChanged = true;
-                  }),
-                ),
-              ],
-            ),
-          ),
+                Column(children: [
+                  NumberPicker(
+                    value: _exerciseTime,
+                    minValue: 1,
+                    maxValue: 50,
```

Comment:
```suggestion
                    maxValue: 120,
```
NumberPicker maxValue should match the max value of the element below